api_session.yaml

```
@@ -85,6 +114,13 @@ definitions:
     properties:
       message:
         type: string
+  UserSession:
```

What if we rename this object with
 ```suggestion
   UserSessionStatus:
```
?

---

I named it `UserSession` becouse in the future we could expand it with other fields

---

Nice, maybe could be better `UserSessionInfo`

---

you are right. I'll fix it. Thank you!